platform/podjs/src/browserPod.ts

```diff
@@ -15,68 +15,130 @@ import * as zip from "@zip.js/zip.js";
 import endpointsJson from "../../../../polyPod-config/endpoints.json";
 import { Manifest, readManifest } from "./manifest";
 
+const DB_NAME = "polypod";
+const DB_VERSION = 1;
+const OBJECT_STORE_POLY_IN = "poly-in";
+const OBJECT_STORE_POLY_OUT = "poly-out";
+
 const NAV_FRAME_ID = "polyNavFrame";
 const NAV_DEFAULT_BACKGROUND_COLOR = "#ffffff";
 const NAV_DARK_FOREGROUND_COLOR = "#000000";
 const NAV_LIGHT_FOREGROUND_COLOR = "#ffffff";
 
-class LocalStoragePolyIn implements PolyIn {
-    private static readonly storageKey = "polyInStore";
-    private store = JSON.parse(
-        localStorage.getItem(LocalStoragePolyIn.storageKey) || "[]"
-    );
+async function migrateData(db: IDBDatabase): Promise<IDBDatabase> {
+    const polyInData = localStorage.getItem("polyInStore");
+    const polyOutData = localStorage.getItem("files");
+
+    if (polyInData || polyOutData) {
+        const quads = polyInData ? JSON.parse(polyInData) : [];
```

Comment:
OK, this implies that polyInData is a JSON string. Maybe we should force some schema on it?
Again, this is carried over from the original implementation, but we weren't using it (or needing it) heavily as we do now.